src/VisualStudio/IntegrationTest/IntegrationTestBuildProject.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.Build.Traversal">
+  <PropertyGroup>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+  </PropertyGroup>
+  <!--
+    This build project is used to allow CI to only build the projects required to build and run integration tests.
+    Everything else (for example other unit test projects) are excluded to speed up the build time.
+    -->
+  <ItemGroup>
```

Comment:
It feels like we should be able to eliminate more of these in favor of additional project-to-project references between these projects -- for example I'm not sure why the integration tests don't directly depend on test setup, etc. But this is a far more manageable list, and it seems like each entry here at least raises good questions for follow-up.

Reply:
These are basically just all the VSIX projects + integration test projects + tool projects.  

I'm not sure if the integration test project should reference VSIX projects or not (IntegrationTest.Setup is a VSIX project) - I didn't want to make that change in this PR